README.md

```diff
@@ -178,17 +179,30 @@ If you try to sign-in before the tenant administrator has provisioned the app in
 ![AADSTS700016, App not found](./ReadmeFiles/AADSTS700016.JPG)
 
 ## National Cloud Deviations
-
-In order to run this sample on a National Cloud, you can follow through the steps above with a few variations:
-
-1. You must register this sample for your Azure AD Tenant in a [National Clouds](https://docs.microsoft.com/en-us/azure/active-directory/develop/authentication-national-cloud) by following [[Step 2]](#step-2-register-the-sample-with-your-azure-active-directory-tenant) above in the National cloud of your choice.
- If you using automation provided via Powershell to create your app, you need to change the [Configure.ps1](./AppCreationScripts/Configure.ps1) and [Cleanup.ps1](./AppCreationScripts/Cleanup.ps1) as instructed below to append the `-AzureEnvironmentName` parameter. The details on this parameter and its possible values are listed in [Connect-AzureAD](https://docs.microsoft.com/en-us/powershell/module/azuread/connect-azuread?view=azureadps-2.0).
-
- ```Powershell
- Connect-AzureAD -TenantId $tenantId -AzureEnvironmentName AzureUSGovernment
- ```
-
-1. Then follow the steps outlined in [[Steps 3]](#step-3--configure-the-sample-to-use-your-azure-ad-tenant) above and additionally make the following changes in the `TodoListWebApp\Web.Config` file.
+Microsoft cloud services are also available in three separate national clouds. These national cloud versions are physical and logical network-isolated instances of Microsoft enterprise cloud services that are confined within the geographic borders of specific countries and operated by local personnel.
```

Comment:
In the Overview at the top of the document, Kalyan has done a summary of multi tenancy. I wonder if we would not want to have a summary of sovereign clouds as well there (2 sections)
lines 182-186

Reply:
Again, note my earlier comment on staying on topic relevant to the sample, the deleted note had a link for the national cloud page. I suggest we start with 
"National cloud are physical and logical network-isolated instances of Microsoft Azure that are confined within the geographic borders of specific countries and operated by local personnel. For more details, visit [link]" 

followed by

"In order to run this sample on a National Cloud, you can follow through the steps above with a few variations:"